hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/spi/ContainerDBServiceProvider.java

```diff
@@ -70,13 +70,23 @@ Integer getCountForForContainerKeyPrefix(
       throws IOException;
 
   /**
-   * Get a Map of containerID, containerMetadata of all Containers.
+   * Get a Map of containerID, containerMetadata of all the Containers.
    *
    * @return Map of containerID -> containerMetadata.
    * @throws IOException
    */
   Map<Long, ContainerMetadata> getContainers() throws IOException;
```

Comment:
Minor: I see the only new method being used.
Old method is not being used from the API.

Reply:
We can remove this method, in your further jira for considering the start parameter.